### PR TITLE
pin cupy to < 7.2.0a0

### PR DESCRIPTION
**Pull request by quasiben**
cc @jakirkham @pentschev

### CODE REVIEW

**Comment by jakirkham** [comment]
I think that is ok. We will just get 7.1.1 then.

**Comment by raydouglass** [comment]
# [Codecov](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=h1) Report
> Merging [#239](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=desc) into [branch-0.13](https://codecov.io/gh/rapidsai/dask-cuda/commit/474108a3b597d27339a019a200bd824f6f85ef50&el=desc) will **decrease** coverage by `0.10%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=tree)

```diff
@@               Coverage Diff               @@
##           branch-0.13     #239      +/-   ##
===============================================
- Coverage        76.39%   76.29%   -0.11%     
===============================================
  Files               14       14              
  Lines              983      983              
===============================================
- Hits               751      750       -1     
- Misses             232      233       +1     
```


| [Impacted Files](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [dask\_cuda/device\_host\_file.py](https://codecov.io/gh/rapidsai/dask-cuda/pull/239/diff?src=pr&el=tree#diff-ZGFza19jdWRhL2RldmljZV9ob3N0X2ZpbGUucHk=) | `93.02% <0.00%> (-1.17%)` | :arrow_down: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=footer). Last update [474108a...d8ab37c](https://codecov.io/gh/rapidsai/dask-cuda/pull/239?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ conda list
 # Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
 # indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
 conda install "cudatoolkit=$CUDA_REL" \
-              "cupy>=6.5.0" "numpy=1.16.4" \
+              "cupy<7.2" "numpy=1.16.4" \
```
Comment (jakirkham) [inline ci/gpu/build.sh]: Would add the `a` bit to avoid getting pre-releases as well.

```suggestion
              "cupy<7.2.0a0" "numpy=1.16.4" \
```

**Comment by quasiben** [comment]
@dillon-cullinan what do you think about the new changes?

File: ci/gpu/build.sh [hunk 2]
```diff
@@ -57,7 +57,7 @@ conda list
 # Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
 # indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
 conda install "cudatoolkit=$CUDA_REL" \
-              "cupy>=6.5.0" "numpy=1.16.4" \
+              "cupy<7.2.0a0" "numpy=1.16.4" \
```
Comment (raydouglass) [inline ci/gpu/build.sh]: Any reason why not `cupy>=6.5.0,<7.2.0a0` Also, we have `!=7.1.0` in some other repos.

Comment (quasiben) [reply ci/gpu/build.sh]: I was being lazy with edits.  I'll change that now

Comment (quasiben) [reply ci/gpu/build.sh]: resolved in https://github.com/rapidsai/dask-cuda/pull/239/commits/d33b908c81bc82a8dc536b4c51f05b8bd67ba74c

**Comment by quasiben** [comment]
rerun tests

**Comment by jakirkham** [comment]
Thanks Ben! 😄

**Pull request merged**